`rbw gen-completions bash` breaks when the package was built with a debug
profile.  Upstreamed in 80a5651b98c3e2d1914c7099e5c3b4ac235d3c52.

Bug: https://bugs.gentoo.org/935517
Author: Lucio Sauer <watermanpaint@posteo.net>
--- a/src/bin/rbw/main.rs
+++ b/src/bin/rbw/main.rs
@@ -145,8 +145,8 @@ enum Opt {
             password to the database.",
         visible_alias = "gen",
         group = clap::ArgGroup::new("password-type").args(&[
-            "no-symbols",
-            "only-numbers",
+            "no_symbols",
+            "only_numbers",
             "nonconfusables",
             "diceware",
         ])
